### PR TITLE
Feature#6 create login screen and navigation

### PR DESCRIPTION
**Pull request by Lucas-aquiles**
# Description
-The login screen is created.
- Field validations are added.
-Added navigation between components (login-sign up).
 

  ## Screenshots 
https://user-images.githubusercontent.com/49735490/189773965-38f88b79-d909-4db4-8ee6-6c9335bbc7e9.mp4

### CODE REVIEW

File: app-vuelos/src/views/LoginScreen.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[a-zA-Z]).{8,}$/
+      )
+    ) {
+      handleError("*Please input a valid password", "password");
+      isValid = false;
+    }
+
+    if (isValid) {
+      // register();
+      console.log(alert("user Loggin"));
```
Comment (YilKanda) [inline app-vuelos/src/views/LoginScreen.js]: Do not leave debug code on GitHub.

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Review by TonyHdez-Pi — APPROVED** [review]
good job 👍🏼

**Review by Astroyayo — APPROVED** [review]
All good, now you have the blessing from the almighty 🤠

File: app-vuelos/src/views/LoginScreen.js (hunk 1 above)
Comment (Lucas-aquiles) [reply app-vuelos/src/views/LoginScreen.js]: resuelto

**Pull request merged**